README.md

```diff
@@ -10,25 +16,57 @@ composer require adt/presenter-test-coverage
 ```
 # app/config/config.neon
 extensions:
-    presenterTestCoverage: ADT\PresenterTestCoverage\DI\PresenterTestCoverageExtension
+    componentTestCoverage: ADT\PresenterTestCoverage\DI\ComponentTestCoverageExtension
 ```
 
 ### 1.2 Nastavení
 ```
 # app/config/config.neon
-presenterTestCoverage:
-    appNamespacePrefix: App
-    crawlerNamespacePrefix: Crawler
-    presenterDir: %appDir%/Modules
-    tempDir: %tempDir%
-    testDir: %appDir%/../tests/Crawler
+componentTestCoverage:
+    tempDir: %appDir%/../temp
+    
+    # Složka s akceptačními testy. V této složce se kontroluje, zda nechybí nějaké testy oproti složkám s aplikací, které se definují dále.
+    # V této složce mohou být testy navíc, to ničemu nevadí. Struktura v této složce musí odpovídat struktuře ve složkách s aplikací.
+    testDir: %appDir%/../tests/Acceptance
+    
+    componentCoverage:
+        clientModulePresenters:    # Název kategorie testovaných věcí. Libovolný název.
+            componentDir: %appDir%/Modules/ClientModule    # Složka s implementací daných komponent
+            appFileMask: (.*)Presenter.php    # Regulární výraz: pro které soubory v app složce se má kontrolovat existence testů
+            testFileMask: {$1}PresenterCest.php    # Maska souborů v test složce, pokud testovací framework potřebuje speciální suffix. Např.: Pokud je presenter v `%appDir%/Modules/ClientModule/ClientPresenter.php`, bude se k němu hledat soubor s testem v `<testDir>/Modules/ClientModule/ClientPresenterCest.php`.
+            methodMask: action.*    # Regulární výraz: pro které metody se mají hledat metody v testech. Aktuálně se jména musí shodovat 1:1.
+        grids:
+            componentDir: %appDir%/Components/Grids
+            appFileMask: (.*)Grid.php
+            testFileMask: {$1}GridCest.php
+            methodMask: render.*
+	...
+```
+
+
+## 2 Spuštění commandu
+```
+$ php bin/console adt:component-test-coverage
 ```
 
-### 1.3 Příklad testované třídy
+
+### 2.1 Přepínače
 ```
-namespace App\Modules\ClientModule\Diary;
+--PSR4 - provede kontrolu zda se neyskytují soubory, které nejsou kompatibilni s PSR4 standardem pokdu se vyskytnou je vrácena hodnota 1. Na obrazovce se vypíše seznam souborů neplnících PSR-4.
 
-class DiaryPresenter extends BasePresenter
+--MissingTests - provede kontrolu zda se v projektu nenecházejí metody pro které nejsou vytvořeny testy. Pokud jsou takové nalezeny, je vrácena hodnota 1 a na obrazovce se vypíší chybějící testy.
+
+WrongConfiguration - zjistí zda není chyba v konfiguraci. Pokud existuje chyba, vrátí hodnotu 1.
```

Comment:
Chybí `--`

Reply:
A jaký je rozdíl oproti MissingTests? Z implementace se mi na první pohled zdá, že je to úplně to stejné.

Reply:
wrong-configuration postrádalo po nasazení nette/scheme smysl, přepínač, getter a příslušná proměnná byli odstraněni.